dds_web/commands.py

```diff
@@ -730,6 +731,8 @@ def collect_stats():
         # Amount of data
         bytes_stored_now: int = sum(proj.size for proj in Project.query.filter_by(is_active=True))
         tb_stored_now: float = round(bytes_stored_now / 1e12, 2)
+        bytes_stored_since_start = sum(version.size_stored for version in Version.query.all())
```

Comment:
Since there are going to be a lot of versions I think we should use the page_query function.
```suggestion
        bytes_stored_since_start = sum(version.size_stored for version in page_query(Version.query))
```

Reply:
yes indeed, there's going to be huge amount of versions and this is basically for optimizing memory usage for such a query!